### PR TITLE
Add a scotland poster image to images

### LINKED ISSUE #3928: Add a Scotland Poster Image
**Is your feature request related to a problem? Please describe.**
For the /scotland service we need a deafult poster image for Opengraph, Twitter, Schema.org etc.

**Describe the solution you'd like**
Create an image and make it available at https://news.files.bbci.co.uk/include/articles/public/scotland/images/og/poster-1024x576.png, and update the file path in the service config

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #3928

**Overall change:** 
Adds a poster image for /scotland

**Code changes:**

- Add a directory in /public/scotland/images called og (for OpenGraph)
- Update the /scotland config with the new image URL

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/scotland.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const service = {
     brandName: 'BBC Scotland',
     product: 'BBC Scotland',
     defaultImage:
-      'https://www.bbc.co.uk/news/special/2015/newsspec_10857/bbc_news_logo.png',
+      'https://news.files.bbci.co.uk/include/articles/public/scotland/images/og/poster-1024x576.png',
```
Comment (HarveyPeachey) [inline src/app/lib/config/services/scotland.js:25]: Am I right in thinking that when this gets merged this URL will resolve itself?

Comment (jimjohnsonrollings) [reply src/app/lib/config/services/scotland.js:25]: > Am I right in thinking that when this gets merged this URL will resolve itself?

That was my thinking...

**Review by simonsinclair — APPROVED** [review]
The poster image looks good. It has the correct dimensions, none of the border issues we were having before and is optimised.

I performed some manual local testing to ensure the new `defaultImage` URL was being used and it is - see below.

Screen Shot 2019-09-23 at 14 58 10

Assuming the URL will resolve, this looks good to me.

**Review by sareh — APPROVED** [review]
:+1: Looks good!

**Pull request merged**